settings.gradle

```diff
@@ -153,3 +153,6 @@ if (extraProjects.exists()) {
     addSubProjects('', extraProjectDir)
   }
 }
+include 'plugins:custom-codecs'
```

Comment:
Not needed, covered by `addSubProjects('', new File(rootProject.projectDir, 'plugins'))` above